plugin/README.md

```diff
@@ -1,15 +1,15 @@
 # Verifiable Credential Service Worker Plugin for ReSpec
 
-But with support for v2, and no JSON-LD processing.
+Builds upon [respec-vc](https://github.com/w3c/respec-vc), but with support for v2, and no JSON-LD processing.
```

Comment:
`[[[VCDM v2]]] [[VCDM v2]]` should get the right citation link, but I don't immediately know the `[[shortcut]]`.
```suggestion
Builds upon [`respec-vc`](https://github.com/w3c/respec-vc), adding support for [[[VCDM v2]]] [[VCDM v2]], and removing JSON-LD processing.
```

Reply:
ah this is plain markdown not respect, so will need the real links. I will add them. thank you